### ISSUE TITLE
Sémaphores - Modifier un contact mail et un nom de sémaphore

### ISSUE THREAD

**Issue by TEAMCACEM** [MTES-MCT/monitorenv#607]
St Quay : semaphore-saint-quay.cdq.fct@intradef.gouv.fr est la bonne adresse actuellement c'est : semaphore saintquay.cdq.fct@intradef.gouv.fr

**Comment by AdelineCelier** [comment]
Zut pardon je n'avais pas vu ce ticket. 
C'est noté, on le traitera à la prochaine mise en prod.

**Comment by TEAMCACEM** [comment]
Parata : semaphore-la-parata.cdq.fct@intradef.gouv.fr est la bonne adresse mail
Sémaphore d'Alisto. Modifier pour Sémaphore d'Alistro (avec un R)
Question de Christophe : modifier le nom du sémaphore modifiera-t-il les données des précédentes missions liées à cette unité ?

**Comment by AdelineCelier** [comment]
Hello la @TEAMCACEM 
Alors je me rends compte que je me suis trompée dans ma première réponse : je crois bien que c'est vous qui avez la main sur les mails des sémaphores via QGIS, je pense qu'Alexia tu peux directement y rectifier les coquilles des mails. Désolée d'avoir autant tardé à m'en rendre compte. 

 Concernant la coquille dans le nom du sémaphore d'Alistro : 
- vous pouvez la corriger direct dans QGIS, cela réglera le problème pour le sémaphore affiché sur la carte et n'impactera pas les données, puisqu'on ne fait pas de stats sur ce référentiel QGIS
- c'est à nous de la corriger dans la liste des unités à la prochaine mise en prod. Normalement cela ne devrait pas impacter le suivi des données enregistrées sur cette unité, mais on attendra la confirmation de @thoomasbro et @VincentAntoine avant de le faire ;)

**Comment by AdelineCelier** [comment]
Hello le CACEM, 
Nous avons rajouté un "r" à Alistro dans la liste des unités.
Comme dit ci-dessus, c'est vous qui avez la main pour modifier le nom et le contact du sémaphore dans QGIS, pour les signalements. (A terme les sémaphores seront retirés de la liste des unités, et il n'y aura à les gérer que dans QGIS pour les signalements).

**Comment by TEAMCACEM** [comment]
(Roxane) il manque encore un - pour le mail de semaphore**-**saintquay

**Comment by AdelineCelier** [comment]
Oui, mais c'est dans QGIS qu'il faut le rajouter, c'est de votre côté que ça se passe.

**Comment by TEAMCACEM** [comment]
(Alexia) Mince, je n'avais pas vu passer ces messages. C'est modifié.
